### ISSUE TITLE
Original power menu and new buttons not hidden in release 45

### ISSUE THREAD

**Issue by internauta2000** [PRATAP-KUMAR/bring-out-submenu-of-power-off-logout#38]
Original power menu and new buttons not hidden in last version, the configuration options not change nothing, only show new buttons in gnome 45.1 Manjaro

**Comment by pratap-panabaka** [comment]
Hi, I will look into it. Meanwhile can you try with the repository version via Makefile? It is updated one.

Thanks!

**Comment by pratap-panabaka** [comment]
`    <schema id="org.gnome.shell.extensions.bring-out-submenu-of-power-off-logout" path="/org/gnome/shell/extensions/org.gnome.shell.extensions.bring-out-submenu-of-power-off-logout/">`

Line number 2 in `schemas/org.gnome.shell.extensions.bring-out-submenu-of-power-off-logout.gschema.xml` this file must be the cause in official version of this extenion. I will fix it. but for time being can you go with Makefile as mentioned above?

**Comment by internauta2000** [comment]
The update does not change unfortunately, the problem is there...

![bring-out-submenu-of-power-off-logout](https://github.com/PRATAP-KUMAR/bring-out-submenu-of-power-off-logout/assets/12079105/8b7381e1-be94-4ed5-be8b-3ce261a31086)

![Bring Out Submenu Of Power Off Button 2](https://github.com/PRATAP-KUMAR/bring-out-submenu-of-power-off-logout/assets/12079105/8d46e416-9580-4ec1-9092-e759c15ec9f2)

**Comment by pratap-panabaka** [comment]
Hi can you remove the current extension and then install the extension via the readme provided.
for now can you share the content of extensions `metadata.json` file please.

**Comment by internauta2000** [comment]
metadata.json...

{
  "description": "Bring Out Submenu Of Power Off Button",
  "name": "Bring Out Submenu Of Power Off Button",
  "settings-schema": "org.gnome.shell.extensions.bring-out-submenu-of-power-off-logout",
  "shell-version": [
    "45"
  ],
  "url": "https://github.com/PRATAP-KUMAR/bring-out-submenu-of-power-off-logout",
  "uuid": "bring-out-submenu-of-power-off-logout@pratap.fastmail.fm",
  "donations": {
    "buymeacoffee": "pratappanabaka",
    "paypal": "pratappanabaka"
  },
  "version-name": "51"
}

**Comment by pratap-panabaka** [comment]
OK thanks for the output. Is the problem still there?

**Comment by internauta2000** [comment]
Yes..! The update does not change unfortunately,

**Comment by pratap-panabaka** [comment]
Ok turnoff all extensions first including this. Reboot the system and then turn on only this extension and see please. If still problem exists then I should check for your particular linux distro.

**Comment by internauta2000** [comment]
"User Avatar In Quick Settings" extension stop working when activating Bling and the problem reappears.... I suggest checking the code of that extension.

**Comment by pratap-panabaka** [comment]
Hi,
Bling n UserAvatar??

have you tried turning off all extensions as I mentioned above? What was the result of that please.

**Comment by internauta2000** [comment]
Yes, I deactivated all extensions and reactivate one by one to replicate the problem... UserAvatar is broken when installing Bling, in fact, if I install Bling first and UserAvatar then it never works because Bling does not allow.

**Comment by pratap-panabaka** [comment]
Hi I am confused with `Bling`. By any chance are you referring it to Bring-out extension? I am looking for this, can you recheck this please? 

> Ok turnoff all extensions first including this. Reboot the system and then turn on only this extension and see please. If still problem exists then I should check for your particular linux distro.

**Comment by internauta2000** [comment]
I apologize for error when writing Bling ... I always wanted to say Bring (bring-out-submenu-of-power-off-logout)

**Comment by internauta2000** [comment]
Greetings... The problem raised here reappeared again with its last 56 update, generating the same conflict with User "Avatar In Quick Settings" extension...
I attach the developer's comment:
https://github.com/d-go/quick-settings-avatar/issues/10
![Screenshot-20240616152011-1268x692](https://github.com/PRATAP-KUMAR/bring-out-submenu-of-power-off-logout/assets/12079105/9f18e9e1-6170-4bb6-8cb8-dd1f97876988)

**Comment by pratap-panabaka** [comment]
Hi @internauta2000, I have gone through the links you provided. I tried with the hints given by quick-settings-avatar's extension author. Since both the extensions run at the same time, there seems to be conflict with the items removing and adding by both the extensions.

Can you try to add time out in Quick Settings Avatar's extension?
Around line numbers 211 to 213 the code in QSA `extension.js` file should look like below after adding setTimeout
```
        if (QuickSettingsMenu._system) {
            setTimeout(() => {
                this._indicator = new Indicator(this._mapSettings());
            }, 5000)
        } else {
            sourceId = GLib.idle_add(GLib.PRIORITY_DEFAULT, () => {
                if (!QuickSettingsMenu._system)
                    return GLib.SOURCE_CONTINUE;

                this._indicator = new Indicator(this._mapSettings());
                return GLib.SOURCE_REMOVE;
            });
        }
```
![Screenshot from 2024-06-17 09-35-49](https://github.com/PRATAP-KUMAR/bring-out-submenu-of-power-off-logout/assets/40719899/78c2048d-061d-4912-b077-d20f5723a400)

![Screenshot from 2024-06-17 09-36-18](https://github.com/PRATAP-KUMAR/bring-out-submenu-of-power-off-logout/assets/40719899/860368f4-fb6f-4897-a121-bbee9a43af74)

**Comment by internauta2000** [comment]
Thank you for your prompt response... When I apply the patch the problem disappears, which suggests that it is resolved momentarily.
I am going to try to transmit the solution to the extension developer so that they can collaborate as well... Greetings.

**Comment by pratap-panabaka** [comment]
Nice, you can trail and error with setTimout from 5000 to 4000 or 3000 or 2000 to reduce the delay that works for your machine.

Thank You for your quick response.
